src/components/TextField/index.tsx

```diff
@@ -188,9 +188,13 @@ const TextField: FunctionComponent<TextFieldProps> = forwardRef(
       isValid,
       isValidVisible,
       isInvalidVisible,
+      required,
```

Comment:
you shouldn't need this because you should get got free with 
InputHTMLAttributes<HTMLInputElement> 
which is added to the prop types which is why pulling it out of props.required didn't complain.

It's always best to use the props that you get for free with the element attribute type that you get out of the box, and only add any custom extra props that you need on top.

Reply:
makes sense, remove "required"